File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.js
function solution(a, b) {
    // a ⊕ b: a와 b를 문자열로 붙여서 Number로 변환
    const concat = Number(String(a) + String(b));
    
    // 2 * a * b
    const product = 2 * a * b;
    
    // 더 큰 값 반환, 같으면 concat 반환
    return concat >= product ? concat : product;
}

File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.java
public class Solution {
    public long solution(long a, long b) {
        // a ⊕ b 계산: 두 수를 문자열로 붙인 뒤 long으로 변환
        long concat = Long.parseLong(a + "" + b);
        
        // 2 * a * b 계산
        long product = 2 * a * b;
        
        // 더 큰 값 반환
        // 같으면 concat (a ⊕ b)를 반환
        if (concat >= product) {
            return concat;
        } else {
            return product;
        }
    }
}
